README.md

```diff
@@ -160,8 +160,10 @@ Default directory structure
   - `controllers`: folder for controller files.
   - `models`: folder for model files.
   - `views`: folder for view files.
+- `lib`: Optional folder for your application's utility libraries.
 - `node_modules`: a standard folder where all modules your app depends on (such as Roosevelt) are installed to. This folder is created by the `npm install` command.
   - `models`: symlink to `mvc/models` so you can `require('models/yourModel')` anywhere in your code without specifying a relative path. Roosevelt will create this symlink for you.
+  - `lib`: symlink to `lib` so you can `require('lib/yourLib')` anywhere in your code without using a relative path. If you've implemented a lib directory, Roosevelt will create this symlink.
```

Comment:
Instead of "If you've implemented a lib directory" rewrite that to "If the `lib` folder is present"